### PR TITLE
Cli better back

### LINKED ISSUE #537: CLI: way to go back in all subtasks? Might involve getting rid of the rudimentary back thing.

### PR DESCRIPTION
**Pull request by ajsantander**
Fix #537

It turned out to be very hard to exit prompts with `esc` or `ctrl-c` with inquirer, so I ended up replacing the lib with `prompts`, which seems to be more lightweight and modern. It easily allows you to cancel prompts, and has autocomplete lists components out of the box.

This PR also includes a more elegant solution for how subtasks are linked in the endless interactive loop in the main task.

### CODE REVIEW

File: packages/cli/subtasks/execute-call.js [hunk 1]
```diff
@@ -3,14 +3,23 @@ const { SUBTASK_EXECUTE_CALL } = require('../task-names');
 const logger = require('@synthetixio/core-js/utils/logger');
 const chalk = require('chalk');
 const prompter = require('@synthetixio/core-js/utils/prompter');
+const { getSignatureWithParameterNamesAndValues } = require('../internal/signatures');
 
 subtask(SUBTASK_EXECUTE_CALL, 'Execute the current tx').setAction(async (taskArguments, hre) => {
-  const abi = hre.deployer.deployment.abis[hre.cli.contractName];
-  const functionAbi = abi.find((abiItem) => abiItem.name === hre.cli.functionName);
+  logger.info(
```
Comment (ajsantander) [inline packages/cli/subtasks/execute-call.js:9]: Ended up merging `preview-call` and `execute-call`

File: packages/core-js/package.json [hunk 2]
```diff
@@ -16,12 +16,12 @@
...
   "dependencies": {
     "chalk": "4.1.2",
-    "ethers": "5.5.1",
-    "inquirer": "8.1.2"
+    "ethers": "5.5.1"
   },
   "devDependencies": {
     "mocha": "9.1.1",
     "nyc": "15.1.0",
+    "prompts": "2.4.2",
```
Comment (leomassazza) [inline packages/core-js/package.json:24]: Is it a dev dependency or user dependency?

File: packages/core-js/utils/prompter.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
-const inquirer = require('inquirer');
+const prompts = require('prompts');
 
 module.exports = {
-  _prompt: inquirer.prompt.bind(inquirer),
+  // _prompt: prompts.prompt.bind(prompts),
```
Comment (leomassazza) [inline packages/core-js/utils/prompter.js:4]: rogue comment?

**Review by leomassazza — APPROVED** [review]
with some minor np

File: packages/core-js/utils/prompter.js (hunk 3 above)
Comment (ajsantander) [reply packages/core-js/utils/prompter.js]: Fixed!

File: packages/core-js/package.json (hunk 2 above)
Comment (ajsantander) [reply packages/core-js/package.json:24]: Nice! Should be a user dep.

**Pull request merged**